c10/util/Unicode.h

```diff
@@ -25,5 +25,31 @@ inline std::wstring u8u16(const std::string& str) {
             size_needed);
     return wstr;
 }
+inline std::string u16u8(const std::wstring& wstr) {
```

Comment:
FWIW, probably don't actually want to inline this

Reply:
Fixed.